app/jenkins-for-jira-ui/src/components/JenkinsServerList/ConnectedServer/JenkinsModal.tsx

```diff
@@ -44,35 +51,61 @@ const JenkinsModal: React.FC<ModalProps> = ({
 	secondaryButtonAppearance,
 	secondaryButtonLabel,
 	secondaryButtonOnClick,
-	isLoading
+	isLoading,
+	isCopiedToClipboard
 }: ModalProps): JSX.Element => {
 	return (
 		<ModalTransition>
 			{show && (
 				<Modal onClose={onClose} testId={dataTestId}>
 					<ModalHeader>
-						<ModalTitle appearance={modalAppearance}>{title}</ModalTitle>
+						{
+							modalAppearance
+								? (
+									<ModalTitle appearance={modalAppearance}>{title}</ModalTitle>
```

Comment:
The only 2 modal appears are warning/danger so we only want to apply for some use cases.